devicemaps.json

```diff
@@ -1139,6 +1139,234 @@
           }
         ],
         "lte": []
+      },
+      "SSR1500": {
+        "clei": "INMJY10HRA",
+        "part_number": "650-147539",
+        "ethernet": [
+          {
+            "pciAddress": "0000:41:00.0",
+            "type": "MGMT",
+            "speed": 1000,
+            "name": "mgmt-0-0",
+            "description": "Port MGMT labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "mgmt",
+                "description": "MGMT network interface, connected to port MGMT labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:44:00.0",
+            "type": "WAN",
+            "speed": 1000,
+            "name": "ge-1-0",
+            "description": "Port 1/0 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "wan1",
+                "description": "WAN 1 network interface, connected to port 1/0 labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:44:00.1",
+            "type": "WAN",
+            "speed": 1000,
+            "name": "ge-1-1",
+            "description": "Port 1/1 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "wan2",
+                "description": "WAN 2 network interface, connected to port 1/1 labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:44:00.2",
+            "type": "WAN",
+            "speed": 1000,
+            "name": "ge-1-2",
+            "description": "Port 1/2 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "wan3",
+                "description": "WAN 3 network interface, connected to port 1/2 labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:44:00.3",
+            "type": "HASync",
+            "speed": 1000,
+            "name": "ge-1-3",
+            "description": "Port 1/3 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "ha-sync",
+                "description": "HA sync network interface, connected to port 1/3 labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:42:00.0",
+            "type": "HAFabric",
+            "speed": 10000,
```

Comment:
10G for HAFabric? Is that intentional and does that line up with other similar platforms? Just asking coz it seemed a bit off.

Reply:
LAN and WAN interfaces are also 10+Gbps, therefore HA fabric interface for dogleg traffic needs to be similarly sized.